models.py

```diff
@@ -247,11 +247,15 @@ def __getattr__(self, name):
     elif self.key.kind() == 'Mastodon':
       args = (auth_entity.instance(),) + args
       inst = auth_entity.app.get().instance_info
+      if inst:
+        j = json_loads(inst)
+        truncate_text_length = j.get("configuration", {}).get('statuses', {}).get('max_characters', None) or j.get('max_toot_chars', None)
```

Comment:
minor style nit for the future, you can drop the two `, None`s here

Reply:
sorry, with timezones delays, I could not remove the `None` here in time (as that's the default from `dict.get`, you're right)

Reply:
No worries! I merged it immediately after commenting, I didn't expect you to. Thanks again!